src/status_im/ui/screens/syncing/sheets/sync_device_notice/views.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im.ui.screens.syncing.sheets.sync-device-notice.views
+  (:require
+   [quo.react-native :as rn]
+   [status-im.ui.screens.syncing.sheets.sync-device-notice.styles :as styles]
+   [status-im.ui.screens.syncing.sheets.enter-password.views :as enter-password]
+   [re-frame.core :as re-frame]
+   [status-im.ui.components.react :as react]
+   [quo2.components.buttons.button :as quo2-button]
+   [status-im.utils.handlers :refer [>evt]]
+   [status-im.react-native.resources :as resources]))
+
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide])
+  (re-frame/dispatch event))
+
+(defn views []
+  [:<>
+   [rn/view {:style styles/sync-devices-header}
+    [react/image {:source (resources/get-image :sync-new-device)
+                  :style styles/sync-devices-header-image}]]
+   [rn/view {:style styles/sync-devices-body-container}
+    [rn/text {:style styles/header-text} "Sync a new device"]
```

Comment:
can we not use quo components here. and then configure with props rather than the styles used. 

```
e.g (def synced-devices-text
  {:font-size 13
   :font-weight :500
   :color colors/neutral-40})
   ```

Reply:
hmm this was made 3 months ago, the idea was to swiftly get the UI up and running, I shall refactor :)
And definitely using quo2 text component is the right way to go here

Reply:
fixed in latest commit